common/helpers/__tests__/colors.spec.js

```diff
@@ -0,0 +1,66 @@
+import { campCollaborationColor, idToColor, userColor } from '../colors.js'
+
+describe('idToColor', () => {
+  it.each([
+    [
+      [undefined, false],
+      [0, 0, 30],
+    ],
+    [
+      [null, false],
+      [0, 0, 30],
+    ],
+    [
+      ['', false],
+      [0, 0, 30],
+    ],
+    [
+      ['0000000', false],
+      [0, 100, 30],
+    ],
+    [
+      ['0000000', true],
+      [0, 0, 30],
+    ],
+    [
+      ['fffffff', false],
+      [15, 100, 30],
+    ],
+    [
+      ['Wrong input', false],
+      [0, 100, 30],
+    ],
+  ])('maps %p to %p', (input, expected) => {
+    expect(idToColor(...input).coords).toEqual(expected)
```

Comment:
Or actually, since you always use `.toString({ format: 'hex' })`, why not change the output of the color helpers to already output a hex color string? The color helpers would be way easier to use then, and the tests would automatically have to use hex color strings as well, which would make them more realistic.

Reply:
And while you are at it, why not also move the `new Color` calls inside these helpers. Then, our code could simply keep on using the helpers, and wouldn't have to care about the specific color library which you use internally.

Reply:
I'm not sure. Because it would be a bit redundant to create a new color object and then parse it again, instead of using it again. What I could create is a function `colorToString` inside colors.js to always have the same format.

Reply:
In the code, we never use the Color objects, we only work with the color hex strings, or am I missing something? So also in your implementation, the Color object is just an intermediate object.